CodeStyle/build/PreventLocalPack.targets

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+	<Target Name="_PolyadicCodeStylePreventLocalPack" BeforeTargets="Pack"
+			Condition="'$(ContinuousIntegrationBuild)' != 'true' And '$(IsPackable)' == 'true' And '$(ApiCompatGenerateSuppressionFile)' != 'true'">
+		<PropertyGroup>
+			<_WorkflowUrl>$(PackageProjectUrl)/actions?query=branch%253Amain</_WorkflowUrl>
+			<_LinkPrefix>%1b]8;;</_LinkPrefix>
```

Comment:
what's this wizardry hahaha

Reply:
there's an escape sequence for hyperlinks in terminals: https://gist.github.com/egmontkob/eb114294efbcd5adb1944c9f3cb5feda
don't mind the weird encoding, MSBuild uses percent-encoding for weird characters :D